### DISCUSSION TITLE
When will a new version be released or prereleased?

### DISCUSSION THREAD

**Discussion by kmsFlavio** [reqnroll/Reqnroll#129] [Q&A] [answered]
With the pr #126 was solved also our problem. Now we need a new version of Reqnroll. Could you please tell us when the next version will be released or pre-released?

**Comment by gasparnagy** [comment, marked as answer]
@kmsFlavio latest next week, but if we are lucky then maybe even this week.

**Comment by mcallaghan-geotab** [comment]
definitely interested is giving this a shot! seeing lots of great fixes since the Feb 1.0.1 release

**Reply by gasparnagy** [reply]
@mcallaghan-geotab Yes... we have got into a bad loop by having breaking changes but not having good enough tests to port back smaller bug fixes. I hope we will get out of this now.

**Reply by ajeckmans** [reply]
It is going to be fine @gasparnagy . Just some legacy that needs to be fixed.

**Reply by gasparnagy** [reply]
@ajeckmans I hope I'm not part of that... 😉 Trying not to...

**Comment by gasparnagy** [comment]
It is out: https://github.com/orgs/reqnroll/discussions/142

(closing this topic now)
